packages/node-core/src/subcommands/reindex.service.ts

```diff
@@ -79,6 +78,8 @@ export class ReindexService<P extends ISubqueryProject, DS extends BaseDataSourc
       timestamp,
     };
 
+    this.multiChainRewindService.chainId = this.project.network.chainId;
```

Comment:
Doing this seems to conflict with the `init` method on the multichain rewind service. It doesn't seem very intuitive that this should be used instead of `init`.

Also what happens here if this subcommand is run when there is a rewind already in progress initiated by another chain? A test for this would be good